### PR TITLE
Run libbpfgo self tests on self hosted github action runner

### PR DESCRIPTION
**Pull request by grantseltzer**
Signed-off-by: grantseltzer <grantseltzer@gmail.com>

### CODE REVIEW

File: .github/workflows/pr-libbpfgo.yaml
Comment (simar7) [inline .github/workflows/pr-libbpfgo.yaml:8]: Is this a secret stored somewhere? I'm just curious where the value comes from.

Comment (simar7) [reply .github/workflows/pr-libbpfgo.yaml:8]: Ah nvm https://github.com/aquasecurity/tracee/pull/693/files#diff-b5f15b3b6c9c91e9bede2818583e85ac47b2f7efc6331846a73daf3b566c3578R11 must be this right? 

For my understanding, how does it know where the self hosted machine is located?

Comment (grantseltzer) [reply .github/workflows/pr-libbpfgo.yaml:8]: Yes that is it. That label (centos-8.2-linux-4.18) corresponds to the machine which was setup in the settings of the repo. ([docs here](https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners) if you're curious)

**Comment by grantseltzer** [comment]
I've updated this so that the self hosted actions runners are run manually to avoid security concerns. This also makes sense to have in their own files because I have tests in the pipeline that make sense to only run on self hosted runners.

File: .github/workflows/pr-ebpf-selfhosted.yaml
Comment (simar7) [inline .github/workflows/pr-ebpf-selfhosted.yaml:3]: I assume this is what makes it trigger manually right? I was reading the docs but didn't see the prompt like shown https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/

Will I see it once we merge it?

Comment (simar7) [reply .github/workflows/pr-ebpf-selfhosted.yaml:3]: Also who can trigger this? The maintainers of this repo or anyone in Aqua Github org?

Comment (grantseltzer) [reply .github/workflows/pr-ebpf-selfhosted.yaml:3]: I merged this in my personal fork to confirm, yes it will show once merged:

Screen Shot 2021-04-21 at 4 06 42 PM

And I believe maintainers can trigger it.

Comment (grantseltzer) [reply .github/workflows/pr-ebpf-selfhosted.yaml]: Also I renamed the self hosted workflow files.

**Review by simar7 — APPROVED** [review]
Litmus!

**Pull request merged**
